app/src/amazonWebview/java/org/mozilla/focus/webview/FirefoxWebView.kt

```diff
@@ -35,6 +36,11 @@ internal class FirefoxWebView(
     private val chromeClient: FirefoxWebChromeClient
 ) : NestedWebView(context, attrs), IWebView {
 
+    init {
+        val initialZoom = context.resources.getInteger(R.integer.webview_initial_zoom)
+        setInitialScale(initialZoom)
```

Comment:
As discussed IRL, this makes YouTube a poor experience. A value between 200-250 seems better. Also, I'd like to try other values in WebSettings, especially minimumFontSize (I was experimenting between 24-32, iirc).

Reply:
Hmm, I still feel pretty strongly that we shouldn't be changing any font sizes unless we build out a way for users to adjust them themselves.  I'm trying out 225 zoom and 24 minimum font now, and this makes half of the results on a Google search look really bad.  That might be fine if it could be adjusted as needed, but I'm very uncomfortable with us breaking so much of the web in a way that the user can't control.

A few 225 zoom / 24 minimum font examples
![gs2](https://user-images.githubusercontent.com/13170306/61835420-77052580-ae30-11e9-9977-6b9e1a8115a6.png)
![gs3](https://user-images.githubusercontent.com/13170306/61835427-7ff5f700-ae30-11e9-8162-6a117f11cd21.png)

Reply:
To be fair, building that out probably wouldn't be particularly difficult, depending on the UX requirements.

Reply:
@Baron-Severin Are these examples using both zoom and font size simultaneously? What benefits does it have to declare them both?

> I still feel pretty strongly that we shouldn't be changing any font sizes unless we build out a way for users to adjust them themselves.

We should talk IRL but what are you reasons for this?

In hindsight, I think we should have gotten Bram's opinion yesterday. I wonder if another designer is around and we could get their opinion.